libs/layout/package.json

```diff
@@ -21,7 +21,9 @@
     "@angular/cdk": "^9.0.0",
```

Comment:
Why here are `@angular` modules version 9 ?

Reply:
Package updated to v10.